### ISSUE TITLE
new install fails- bosh director certificate

### ISSUE THREAD

**Issue by engrun** [EngineerBetter/concourse-up#52]
Due to a bug with the 3.14.1. release of concourse, we have taken backups of the credhub secrets and the pipeline configs. 

We ran `concourseup destroy` first, which failed, we fixed the cause, ran `destroy` again, this time ok.

No we are trying to install anew with the same name, however, concourseup 0.9.2 fails with the following.

```
GENERATING BOSH DIRECTOR CERTIFICATE (34.240.151.184, 10.0.0.6)
2018/07/02 15:46:11 [INFO] acme: Registering account for nobody@example.com
acme: Error 400 - urn:acme:error:invalidEmail - Error creating new registration :: invalid contact domain. Contact emails @example.com are forbidden
```

How can we get past this problem?

**Comment by iferunsewe** [comment]
Hi @engrun,

This is due to a change at the LetsEncrypt side of things. We fixed it in 83c0debe8f4efa9601212b3dd005dc02d61b7d9f, but using a version of Concourse-Up with this commit will likely bring in a newer version of Concourse itself.

You'll need to [generate your own cert and use that instead](https://github.com/EngineerBetter/concourse-up#custom-domains).

Which bug in 3.14.1 is causing you pain?

**Comment by engrun** [comment]
Hi. Thanks for your quick reply.  This [issue](https://github.com/concourse/concourse/issues/2300) is bugging us
